Harmony/Public/UtfJsonReaderExtensions.net5.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace HarmonyLib
+{
+	internal static class Utf8JsonReaderExtensions
+	{
+		internal static void SkipPropertyName(this Utf8JsonReader reader)
+		{
+			reader.Read(); //this is the `PropertyName`, skip it
+			reader.Read(); //this is the `value`, making it ready to get
+		}
+
+		internal static void WriteStringArray(this Utf8JsonWriter writer, string propertyName, IEnumerable<string> array)
```

Comment:
Can you find a better name than `array`?

Reply:
I can maybe do one of the following?  I was trying to keep it inline with the existing `WriteStartArray` function in `System.Text.Json`

- [ ] `WriteStringData`
- [ ] `WriteStrings`
- [ ] `WriteStringList`
- [ ] `WriteStringSet`